### PR TITLE
kaufkauflist: 3.3.0 → 4.0.0

### PR DESCRIPTION
**Pull request by auroraanna**
## Description of changes

Updated kaufkauflist.

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) (or backporting [23.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md) and [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Review by momeemt — APPROVED** [review]
LGTM :)

**Comment by auroraanna** [comment]
whoops, just realized i made a mistake in one commit. tried putting permissions-policy in the meta http-equiv but that's not supported

**Comment by auroraanna** [comment]
fixed it.

**Review by nicoonoclaste — APPROVED** [review]

**Review by nicoonoclaste — CHANGES_REQUESTED** [review]
Looks good aside from that
(sorry for the previous erroneous review, misclicked :sweat_smile:)

File: pkgs/applications/misc/kaufkauflist/default.nix [hunk 1]
```diff
@@ -21,17 +21,17 @@ let
...
-  version = "3.3.0";
+  version = "4.0.0";
 
   src = fetchFromGitea {
     domain = "codeberg.org";
     owner = "annaaurora";
     repo = "kaufkauflist";
     rev = "v${version}";
-    hash = "sha256-kqDNA+BALVMrPZleyPxxCyls4VKBzY2MttzO51+Ixo8=";
+    hash = "sha256-NzEiYRLHnZhQTqzpWYTHjt1XjCZpWrk0KK94NFxNRe4=";
```
[the remark this answers is not in the document]
Comment (auroraanna) [reply pkgs/applications/misc/kaufkauflist/default.nix]: *tried to update the wrong hash* ;-;

**Comment by marcusramberg** [comment]
Result of `nixpkgs-review pr 310337` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>kaufkauflist</li>
  </ul>
</details>

**Review by marcusramberg — APPROVED** [review]

**Comment by marcusramberg** [comment]
@nbraud think you will have to reapprove for this to be merged as it's now listed as you requesting changes.

**Review by nicoonoclaste — APPROVED** [review]

**Pull request merged**